pkg/reconciler/serverlessservice/serverlessservice.go

```diff
@@ -243,12 +244,12 @@ func (r *reconciler) reconcilePublicEndpoints(ctx context.Context, sks *netv1alp
 		sks.Status.MarkEndpointsNotReady("CreatingPublicEndpoints")
 		eps, err = r.KubeClientSet.CoreV1().Endpoints(sks.Namespace).Create(resources.MakePublicEndpoints(sks, srcEps))
 		if err != nil {
-			logger.Errorw(fmt.Sprint("Error creating K8s Endpoints:", sn), zap.Error(err))
+			logger.Errorw("Error creating K8s Endpoints: "+sn, zap.Error(err))
 			return err
 		}
 		logger.Info("Created K8s Endpoints: ", sn)
 	} else if err != nil {
-		logger.Errorw(fmt.Sprint("Error getting K8s Endpoints:", sn), zap.Error(err))
+		logger.Errorw("Error getting K8s Endpoints: "+sn, zap.Error(err))
```

Comment:
You are inconsistent about using this style vs:
```suggestion
		logger.Errorw("Error getting K8s Endpoints: ", sn, zap.Error(err))
```